v3/plugins/eclipse/che-theia/7.3.0/meta.yaml

```diff
@@ -55,7 +55,7 @@ spec:
          - name: HOSTED_PLUGIN_HOSTNAME
            value: 0.0.0.0
          - name: HOSTED_PLUGIN_PORT
-           value: 3130
+           value: "3130"
```

Comment:
is it worth updating 7.3.0 when 7.3.1, 7.3.2 and 7.4.0 are live?

Reply:
Checker rejects without patching this ...

```
Checking plugin 'eclipse/che-theia/7.3.0'
        3130 is not of type u'string'
```

Reply:
This is similar issue with https://github.com/eclipse/che-plugin-registry/pull/242 .

Reply:
ah, then +1.